### PR TITLE
Added feature to send actual data to existing Arduino Example (Send Message)

### PR DESCRIPTION
**Pull request by dominpiano**
For this to work, the change made in MCreator Link Mod has to be accepted

### CODE REVIEW

File: src/MCreatorLink.cpp [hunk 1]
```diff
@@ -79,7 +79,10 @@ void MCreatorLinkImpl::loop() {
         analogWrite(pin, val);
       }
     } else if (command.equals("msg")) {
-      event(command, data);
+    	String tempComm = data.substring(0, data.indexOf(":"));
```
Comment (KlemenDEV) [inline src/MCreatorLink.cpp]: This need indexOf check as it can be -1, and in this case only use command, with empty string for data

**Review by KlemenDEV — CHANGES_REQUESTED** [review]

File: src/MCreatorLink.cpp (hunk 1 above)
Comment (dominpiano) [reply src/MCreatorLink.cpp:82]: In case there is no data passed, tempData will be empty automatically, I don't have to do any if statements

**Comment by dominpiano** [comment]
And btw, from pure curiosity, because I'm quite new to GitHub and working with people, can You modify my code?
Cause a while ago You asked me to clean the code, like remove debug messages. And wouldn't be easier for You to do it yourself instead of asking me and then after another commit approving it?

**Comment by KlemenDEV** [comment]
> And btw, from pure curiosity, because I'm quite new to GitHub and working with people, can You modify my code?
> Cause a while ago You asked me to clean the code, like remove debug messages. And wouldn't be easier for You to do it yourself instead of asking me and then after another commit approving it?

Yes, I can. But the unspoken rule is PR author has to fix the code, so they learn how to make a better code in the future, and where improvements ca be made.

**Comment by dominpiano** [comment]
Thanks for information!

**Review by KlemenDEV — APPROVED** [review]

**Pull request merged**